samples/subsys/shell/shell_module/src/main.c

```diff
@@ -220,6 +220,50 @@ static int cmd_logout(const struct shell *shell, size_t argc, char **argv)
 	return 0;
 }
 
+#define CHAR_1 0x18
+#define CHAR_2 0x11
+
+static void bypass_cb(const struct shell *sh, uint8_t *data, size_t len)
+{
+	static uint8_t tail;
```

Comment:
The bypass will be active for all backends at the same time? If not maybe this could be added to shell context?